### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by ostelin** [WoeUSB/WoeUSB#73] [General]
The only program that made me, in Linux, an installation flash drive for Windows 7 with one line in the terminal. Thank you very much.

**Comment by cweiske** [comment]
Thank you for WoeUSB!
I tried a couple of options including formatting the usb drive with gparted, making the partition bootable and extracting the iso onto it, but the Asus laptop only showed a blinking underscore.
WoeUSB was the only software that simply worked.

**Comment by brlin-tw** [comment]
Glad to be helpful.
